src/pages/layer-2.js

```diff
@@ -0,0 +1,1172 @@
+            </p>
+            <OrderedList
+              listData={[
+                <p>
+                  <b>A network of node operators</b> to secure and validate the
+                  network
+                </p>,
+                <p>
+                  <b>A network of block producers</b>
+                </p>,
+                <p>
+                  <b>The blockchain</b> itself and the history of transaction
+                  data
+                </p>,
+                <p>
+                  <b>The consensus mechanism</b> for the network
+                </p>,
+              ]}
+            />
+            <p>
+              Still confused on Ethereum?{" "}
+              <Link to="/what-is-ethereum/">Learn what Ethereum is</Link>.
+            </p>
+          </Flex50>
+        </TwoColumnContent>
+      </LightGrayContent>
+
+      <PaddedContent>
+        <TwoColumnContent>
+          <FlexContainer
+            flexPercent="50"
+            style={{
+              display: "flex",
+              alignContent: "center",
+              justifyContent: "center",
+            }}
+          >
+            <GatsbyImage
+              image={getImage(data.dao)}
+              style={{ width: "100%" }}
+              objectFit="contain"
+            />
+          </FlexContainer>
+          <FlexContainer flexPercent="50">
+            <h2>Why do we need layer 2?</h2>
+            <p>
+              Three desirable properties of a blockchain are that it is{" "}
+              <b>decentralized, secure, and scalable</b>. The{" "}
+              <Link to="https://www.ledger.com/academy/what-is-the-blockchain-trilemma">
+                blockchain trilemma
+              </Link>{" "}
+              states that a simple blockchain architecture can only achieve two
+              out of three. Want a secure and decentralized blockchain? You need
+              to sacrifice scalability.
+            </p>
+            <p>
+              Ethereum has reached the network's current capacity with{" "}
+              <Link to="https://etherscan.io/chart/tx">
+                1+ million transactions per day
+              </Link>
+              and high demand for each of these transactions. The success of
+              Ethereum and the demand to use it has caused gas prices to rise
+              substantially. Therefore the{" "}
+              <Link to="/developers/docs/scaling/">
+                need for scaling solutions
+              </Link>{" "}
+              has increased in demand as well. This is where layer 2 networks
+              come in.
+            </p>
+
+            <h3>Scalability</h3>
+            <p>
+              The main goal of scalability is to increase transaction speed
+              (faster finality) and transaction throughput (higher transactions
+              per second) without sacrificing decentralization or security (more
+              on the <Link to="/upgrades/vision/">Ethereum vision</Link>
+              ).
```

Comment:
We've got this twice now, accidentally.

<img width="781" alt="Screenshot 2022-03-31 at 21 08 06" src="https://user-images.githubusercontent.com/62268199/161140713-cfbb3189-fd07-40d7-bb32-c9236c83125e.png">


```suggestion
              per second) without sacrificing decentralization or security.
```